frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -219,13 +225,36 @@ export const Dashboard = () => {
             <MenuItem value="newest-requests-first">
               Newest requests first
             </MenuItem>
-            <MenuItem value="unmatchable-first">Unmatchable first</MenuItem>
-            <MenuItem value="newly-matchable-first">
-              Newly matchable first
-            </MenuItem>
-            <MenuItem value="matchable-first">Matchable first</MenuItem>
+
             <MenuItem value="classes-a-z">Classes A-Z</MenuItem>
             <MenuItem value="classes-z-a">Classes Z-A</MenuItem>
+          </DropdownSelect>
+        </Box>
+        <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+          <Box
+            sx={{
+              fontWeight: 'bold',
+              color: 'essentials.75',
+              padding: 1,
+              margin: 1,
+            }}
+          >
+            Filter:
+          </Box>
+          <DropdownSelect
+            value={filteredOption}
+            onChange={handleFilterChange}
+            sx={{
+              padding: 0,
+              margin: 0,
+              fontWeight: 'bold',
+            }}
+          >
+            <MenuItem value="no-filter">None</MenuItem>
+            <MenuItem value="unmatchable">Unmatchable</MenuItem>
+            <MenuItem value="newly-matchable">Newly matchable</MenuItem>
+            <MenuItem value="matchable">Matchable</MenuItem>
+
             <MenuItem value="no-check-in-email">
```

Comment:
I recommend putting these in a dictionary and rendering them programmatically with a map and doing the same for the search options.